README.md

```diff
@@ -10,3 +10,24 @@
 ## Licence
 
 Apache 2
+
+## Releasing
+
+We don't use `zest.releaser` right now because of some problems with
+releasing a package that is not at the root of the repo (`jobs/`), and
+because we have no interest in uploading this project to PyPI, but
+this could change if we figure out how.
+
+The current procedure is:
+
+* Bump version in `jobs/setup.py`
+* Update the release date in `jobs/CHANGELOG.rst`
+* `make build-dependencies`
+* `git commit -am "Bump x.y.z"`
+* Open PR, wait for it to become green
+* Merge PR
+* `git tag x.y.z`
+* `git push --tags origin`
+* `make lambda.zip`
+* Add a release on Github with the lambda.zip attached
+* Open a ticket like https://bugzilla.mozilla.org/show_bug.cgi?id=1426340 to get it deployed
```

Comment:
So, the only variable is the tag name? 
A link to create the bug can be made with this as a query string variable. 

See how the "Bugzilla" tab on Crash-stats have links to enter a new bug with lots of the stuff already filled in. 
https://crash-stats.mozilla.com/report/index/19ef17b4-3864-4a55-b9a6-ae6e30180204#tab-bugzilla